src/CsvHelper.Excel/ExcelParser.cs

```diff
@@ -123,7 +124,6 @@ public ExcelParser(Stream stream, string sheetName, CsvConfiguration configurati
             }
 
             Context = new CsvContext(this);
-            _leaveOpen = Configuration.LeaveOpen;
```

Comment:
Removing this here, should result in this (and probably more) constructor(s) to have the option to set this directly instead.